cmd/dockerd/daemon.go

```diff
@@ -349,6 +356,12 @@ func (cli *DaemonCli) reloadConfig() {
 			return
 		}
 
+		if b, err := json.Marshal(cli.Config); err != nil {
+			logrus.Errorf("Cannot formatting configuration in JSON: %s", err)
```

Comment:
I am afraid here format is better than formatting.
How about logrus.Errorf("failed to format configuration in JSON: %v", err) ?